test/fixtures/cookbooks/languages_ruby/recipes/default.rb

```diff
@@ -0,0 +1,4 @@
+include_recipe 'chef-sugar::default'
+include_recipe 'languages::default'
+
+ruby_install '2.1.7'
```

Comment:
We should probably add a separate fixture recipe that exercises a non-default prefix (and an associate serverspec test).